docs/Supported-list.md

```diff
@@ -9,6 +9,7 @@
   * [Resin](http://www.caucho.com/resin-4.0/) 3 (Optional¹)
   * [Resin](http://www.caucho.com/resin-4.0/) 4 (Optional¹)
   * [Jetty Server](http://www.eclipse.org/jetty/) 9
+  * [Undertow](http://undertow.io/)  1.x
```

Comment:
I think the version should  be more detail, such as: 1.4.x and 1.3.x, because I found that the 1.x version has contain 1.0.x ,1.2.x, 1.3.x and the test case has only test in 1.3.x and 1.4.x.